packages/telemetry/src/api.ts

```diff
@@ -63,24 +64,44 @@ export function getTelemetry(app: FirebaseApp = getApp()): Telemetry {
  */
 export function captureError(telemetry: Telemetry, error: unknown): void {
   const logger = telemetry.loggerProvider.getLogger('error-logger');
+
+  const activeSpanContext = trace.getActiveSpan()?.spanContext();
+  const traceAttributes = {} as AnyValueMap;
+  if (telemetry.app.options.projectId && activeSpanContext?.traceId) {
```

Comment:
I don't know much about otel but what does it mean if trace.getActiveSpan() doesn't return anything? If so, or if there's no context, or traceId, it skips this whole block. Does the developer need to know about that? Would they be expecting it and be surprised these traceAttributes aren't being sent? Or did they explicitly have to initialize this so it would be clear why it's not being sent?

Reply:
I think we'll eventually want documentation to support this, or wrap OTel's tracing in our SDK. In the context of Cloud Run though, for example, the trace context is typically populated by default. This was manually verified.